src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +28,61 @@ export const TodoList: React.FC = () => (
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+      {todos.map(todo => {
+        const {
+          id,
+          title,
+          completed,
+        } = todo;
 
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+        return (
+          <tr
+            data-cy="todo"
+            className={cn(
+              { 'has-background-info-light': selectedTodo?.id === id },
+            )}
+            key={id}
+          >
+            <td className="is-vcentered">
+              {id}
+            </td>
+            <td className="is-vcentered">
+              {completed
+                && (
+                  <span className="icon" data-cy="iconCompleted">
+                    <i className="fas fa-check" />
+                  </span>
+                )}
+            </td>
+            <td className="is-vcentered is-expanded">
+              <p className={cn(
+                { 'has-text-danger': !completed },
+                { 'has-text-success': completed },
+              )}
+              >
+                {title}
+              </p>
+            </td>
+            <td className="has-text-right is-vcentered">
+              <button
+                data-cy="selectButton"
+                className="button"
+                type="button"
+                onClick={() => onSelectTodo(todo)}
+              >
+                <span className="icon">
+                  <i className={cn(
+                    'far',
+                    { 'far fa-eye-slash': selectedTodo?.id === id },
+                    { 'far fa-eye': selectedTodo?.id !== id },
+                  )}
+                  />
```

Comment:
You should check this logic, because it sets `far` twice when apply one of classes below
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/107207978/35816d52-9c3e-47cf-90dd-a4e03d2e8548)

Reply:
Thanks.